src/libraries/System.Private.Xml.Linq/tests/misc/XAttribute.cs

```diff
@@ -17,5 +16,73 @@ public void FormattedDate()
             // Ensure we are compatible with the .NET Framework
             Assert.Equal("CreatedTime=\"2018-01-01T12:13:14Z\"", new XAttribute("CreatedTime", new DateTime(2018, 1, 1, 12, 13, 14, DateTimeKind.Utc)).ToString());
         }
+
+        public static IEnumerable<object[]> NumericValuesWithMinusSign()
+        {
+            yield return new object[] { -123 };
+            yield return new object[] { -123f };
+            yield return new object[] { -123L };
+            yield return new object[] { (short)-123 };
+            yield return new object[] { -12.3 };
+            yield return new object[] { -12.3m };
+            yield return new object[] { (sbyte)-123 };
+        }
+
+        [Theory]
+        [MemberData(nameof(NumericValuesWithMinusSign))]
+        public void MinusSignWithDifferentTypeSwedishCulture(object value)
+        {
+            CultureInfo newCulture = null;
+            try
+            {
+                newCulture = new CultureInfo("SE-SW");
```

Comment:
This culture `SE-SW` does not seem to exist. Do you mean `sv-SE` ?

Reply:
Were these tests failing without your fix in place?

Reply:
> This culture `SE-SW` does not seem to exist. Do you mean `sv-SE` ?

That is `SE-SW is worked for me (it `Swedish` i heard), by worked i mean on my local i saw the difference of the minus signs before fix.

> Were these tests failing without your fix in place?

To repro you can comment the cases for `int, long, short and sbyte`, for example: https://github.com/dotnet/runtime/pull/46588/files#diff-b2b608017bdbacf5e4484368d6e1fcb03a9ea3b68f4aece41da253f72826acf9R813

Reply:
Interesting it says `sv-SE` is Swedish https://www.localeplanet.com/dotnet/sv-SE/index.html, maybe it is different for windows and linux, not sure how it is worked on my machine

Reply:
What does new CultureInfo("SE-SW").DisplayName give on your machine?

If it's not Swedish, any idea how the tests were failing before your fix?

Reply:
> What does new CultureInfo("SE-SW").DisplayName give on your machine

`davvisámegiella (SW)` it says it is `Sami` language. 

> If it's not Swedish, any idea how the tests were failing before your fix?

Even if it is not Swedish seems this language had similar issue, I tried with Swedish, same char found for minus sign (`\u2212` instead of `\u002d`)

Reply:
Changed test cases into Swedish culture as it reproduces the issue same as `davvisámegiella (SW)` and it might be more popular locale